Libraries/Utilities/Appearance.js

```diff
@@ -85,6 +85,22 @@ module.exports = {
     return nativeColorScheme;
   },
 
+  setColorScheme(colorScheme: string): void {
```

Comment:
This should accept `ColorSchemeName` as the parameter type to limit to only allowed schemes.